<launch>

	<arg name="veh" default="$(env VEHICLE_NAME)"/>
	<arg name="demo_name" value="Test"/>
	<arg name="node_name" default="Controller"/>

	<include file="$(find duckietown_demos)/launch/master.launch">

		<arg name="veh" value="$(arg veh)"/>
        	<arg name="demo_name" value="$(arg demo_name)"/>
		<arg name="param_file_name" value="default" />
		<arg name="visualization" value="false" />
		<arg name="anti_instagram" value="true" />

		<arg name="lane_following" value="true"/>
	    	<arg name="/lane_following/line_detection" value="true"/>
	    	<arg name="line_detector_param_file_name" value="default" />
	    	<arg name="/lane_following/ground_projection" value="true"/>
	    	<arg name="/lane_following/lane_filter" value="false"/>
	    	<arg name="/lane_following/lane_controller" value="false"/>

	</include>

	<group ns="$(arg veh)">

		<!-- <remap from="fakebot/sim_node/lane_pose" to="Disturbance/pose_in"/> -->
		<remap from="Controller/pose" to="Disturbance/pose_out"/>
		<remap from="Controller/cmd" to="wheels_driver_node/wheels_cmd"/>

  		<node pkg="my_package" type="realPID.py" name="$(arg node_name)" output="screen"/>
		<node pkg="my_package" type="disturbance.py" name="Disturbance" output="screen"/>

	</group>
  
</launch>
